api_views/users.py

```diff
@@ -206,3 +206,49 @@ def delete_user(username):
                 return Response(error_message_helper("User not found!"), 404, mimetype="application/json")
         else:
             return Response(error_message_helper("Only Admins may delete users!"), 401, mimetype="application/json")
+
+
+def get_user_by_email(email):
+    user = User.query.filter_by(email=email).first()
```

Comment:
Consider adding input validation for email parameter to ensure proper email format before querying.